metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/BaseJcrProvider.java

```diff
@@ -458,15 +510,15 @@ protected void appendOffset(StringBuilder bldr, Pageable pageable) {
 
     protected void appendSort(StringBuilder bldr, Pageable pageable) {
         boolean first = true;
-        
+
         if (pageable.getSort() != null && pageable.getSort().iterator().hasNext()) {
             bldr.append("ORDER BY ");
             for (Sort.Order order : pageable.getSort()) {
-                if (! first) {
+                if (!first) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2583)